Upstream forgot to update the project version for the v1.2.0 release...

REMOVE THIS PATCH ON NEXT UPDATE

--- CMakeLists.txt.orig	2024-06-20 02:00:04
+++ CMakeLists.txt	2026-09-24 11:47:37
@@ -2,7 +2,7 @@
 
 # set project name
 project(cargs
-  VERSION 1.1.0
+  VERSION 1.2.0
   DESCRIPTION "A simple argument parser library"
   HOMEPAGE_URL "https://likle.github.io/cargs/"
   LANGUAGES C)
